packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -535,6 +577,31 @@ ${REFRESH_UI}
     commentBody
   );
 
+  // Status checks for missing region tag prefix
+  if (
+    configuration.alwaysCreateStatusCheck() ||
+    productPrefixViolations.length > 0
+  ) {
+    try {
+      await context.octokit.checks.create(prefixCheckParams);
+    } catch (e) {
+      e.message = `Error creating prefix status check: ${e.message}`;
```

Comment:
If we were to allow this to throw, and the job was retried, would it result in a comment being placed on a PR twice?

Since we don't expect exceptions to happen, my temptation is usually to allow the process to crash, so that we catch bugs faster.

Reply:
> If we were to allow this to throw, and the job was retried, would it result in a comment being placed on a PR twice?
No, we always re-use the same comment.

> Since we don't expect exceptions to happen, my temptation is usually to allow the process to crash, so that we catch bugs faster.

I can remote `try`.

Reply:
It turned out the `try` swallowed the nock errors.